tests/unit/oidc/test_tasks.py

```diff
@@ -0,0 +1,42 @@
+import pretend
+
+from warehouse.oidc.tasks import compute_oidc_metrics
+
+from ...common.db.oidc import GitHubPublisherFactory
+from ...common.db.packaging import ProjectFactory
+
+
+def test_compute_oidc_metrics(db_request, monkeypatch):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Looks like `monkeypatch` fixture isn't used in the test, should it be removed?
```suggestion
def test_compute_oidc_metrics(db_request):
```